app/models/Amounts.scala

```diff
@@ -11,11 +11,24 @@ case class ContributionAmounts(
     ANNUAL: AmountsSelection
 )
 
-case class AmountsTestVariant(name: String, amounts: ContributionAmounts)
+case class AmountsTestVariant(
+    name: String,
+    amounts: ContributionAmounts,
+    hideChooseYourAmount: Option[Boolean]
+)
 
-case class AmountsTest(name: String, isLive: Boolean, variants: List[AmountsTestVariant], seed: Int)
+case class AmountsTest(
+    name: String,
+    isLive: Boolean,
+    variants: List[AmountsTestVariant],
+    seed: Int
+)
 
-case class ConfiguredRegionAmounts(control: ContributionAmounts, test: Option[AmountsTest])
+case class ConfiguredRegionAmounts(
+    control: ContributionAmounts,
+    test: Option[AmountsTest],
+    hideChooseYourAmount: Option[Boolean]
```

Comment:
did you consider putting the new field in the `ContributionAmounts` class? It would mean only defining it once, alongside the amounts

Reply:
although looking at the typescript definition of `ContributionAmounts` that might be a bit of a nightmare!

Reply:
Funny story - we've ended up at this solution following the Marketing change request ... and I think it simplifies things!